is_pto/preprocess_message.py

```diff
@@ -15,10 +15,10 @@ def clean_sqs_skype_formatting(message):
     cleaned_message = re.sub(r'\B@\w+', '', cleaned_message) #@mentions
     cleaned_message = re.sub(r'&.*?;', '', cleaned_message) #encoded strings
     cleaned_message = re.sub(r'^(\s)*$\n', '', cleaned_message) #emtpy lines
+    cleaned_message = re.sub(r'((?<![\\])["])((?:.(?!(?<![\\])\1))*.?)\1', '', cleaned_message) #whole quoted message
```

Comment:
@nilaya123 - can you please add some tests showing your fix works? I have no idea of how you tested that this fix works.

Reply:
@qxf2 : I have tested it as below:

1. I have tried the quoted message as "I am on PTO today" take care... It gets detected as not a PTO message.
2. Without quotes I have tried the same and it works fine.
![verification is_pto](https://user-images.githubusercontent.com/30227077/111577866-0cf57d00-87d9-11eb-8595-9d4caa5c2f18.JPG)